OpenRA.Mods.Common/Activities/Resupply.cs

```diff
@@ -37,21 +36,24 @@ public class Resupply : Activity
 		readonly bool wasRepaired;
 		readonly PlayerResources playerResources;
 		readonly int unitCost;
+		readonly Actor self;
 
 		int remainingTicks;
 		bool played;
 		bool actualResupplyStarted;
 		ResupplyType activeResupplyTypes = ResupplyType.None;
 
-		public Resupply(Actor self, Actor host, WDist closeEnough, bool stayOnResupplier = false)
+		public Resupply(Actor self, Actor host, bool stayOnResupplier = false)
 		{
+			this.self = self;
 			this.host = Target.FromActor(host);
-			this.closeEnough = closeEnough;
 			this.stayOnResupplier = stayOnResupplier;
 			allRepairsUnits = host.TraitsImplementing<RepairsUnits>().ToArray();
 			health = self.TraitOrDefault<IHealth>();
 			repairable = self.TraitOrDefault<Repairable>();
-			repairableNear = self.TraitOrDefault<RepairableNear>();
+
+			// HACK: Ressuply needs the actor to move to host center, which might not be the centre cell
```

Comment:
I understand this comment is related to a discussion on another changed line, but it honestly doesn't tell me anything. How does the magic fallback value below relate to the host's center? Why is it considered a hack?

Reply:
https://github.com/OpenRA/OpenRA/pull/20169#discussion_r1071414132